.github/workflows/build.yml

```diff
@@ -6,9 +6,12 @@ on:
     branches:
       - master
 env:
-  APP_REPOSITORY:  dfedigital/get-teacher-training-adviser-service
-  PROJECT:         Teacher Training Adviser Service
-  DOMAIN:          london.cloudapps.digital
+  DOCKERHUB_REPOSITORY: dfedigital/get-teacher-training-adviser-service
+  DOMAIN:               london.cloudapps.digital
+  PROJECT:              Teacher Training Adviser Service
```

Comment:
Project has a very specific meaning... Can we use something like `application`, `app`, `app_name`, `service`...

Reply:
Renamed to APPLICATION